src/include/plan_node/abstract_join_plan_node.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <memory>
+#include <utility>
+#include "parser/expression/abstract_expression.h"
+#include "plan_node/abstract_plan_node.h"
+
+namespace terrier::plan_node {
```

Comment:
I don't think naming convention permits `_` in namespaces?

Reply:
Correct.

Reply:
Where is the prohibition against _ in namespaces stated? Because I'm not seeing it.